src/redux/events/eventsSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const FetchEvents = createAsyncThunk(
+  'Country/FetchEvents',
+
+  async (_, thunkAPI) => {
+    const url = 'https://app.ticketmaster.com/discovery/v2/events.json?size=6&countryCode=CA&classificationName=Professional&apikey=7KBAIQZR5JiH4pE3CMdHPToUvYySPND8';
+    try {
+      const resp = await fetch(url);
+      return resp.json();
+    } catch (error) {
+      return thunkAPI.rejectWithValue('something went wrong');
+    }
+  },
+
+);
+
+const initialState = {
+  events: [],
+  isLoading: false,
+};
+
+const EventSlice = createSlice({
+  name: 'events',
+  initialState,
+  reducers: [],
+  extraReducers: (builder) => {
+    builder
+      .addCase(FetchEvents.pending, (state) => {
+        state.isLoading = true;
+      })
+      .addCase(FetchEvents.fulfilled, (state, action) => {
+        state.isLoading = false;
+        // eslint-disable-next-line no-underscore-dangle
+        const data = action.payload._embedded.events;
```

Comment:
- [ ] It will be better 😃 if you not disabling the linter as you you disabled the linter here and that is not the best practice as you know it will be better if you let the linter work as it makes you write clean code without errors, so please try to remove this line to let the linter work and follow the best practice of clean code without errors😊💯